MANIFEST.in

```diff
@@ -1,9 +1,12 @@
 include *.py
 include *.txt
+include *.rst
+include .coveragerc
+include tox.ini
```

Comment:
Downstream distributors (e.g., linux distros, probably conda, etc) do run tests from sdists. I've had a number of complaints from gevent distributors about the test suite it ships.